wled00/bus_manager.h

```diff
@@ -203,10 +203,12 @@ class Bus {
       ww = (w * ww) / 255; //brightness scaling
       cw = (w * cw) / 255;
     }
-    inline        void    setAutoWhiteMode(uint8_t m) { if (m < 5) _autoWhiteMode = m; }
-    inline        uint8_t getAutoWhiteMode()          { return _autoWhiteMode; }
-    inline static void    setGlobalAWMode(uint8_t m)  { if (m < 5) _gAWM = m; else _gAWM = AW_GLOBAL_DISABLED; }
-    inline static uint8_t getGlobalAWMode()           { return _gAWM; }
+    inline        void    setAutoWhiteMode(uint8_t m)     { if (m < 5) _autoWhiteMode = m; }
+    inline        uint8_t getAutoWhiteMode()              { return _autoWhiteMode; }
+    inline static void    setGlobalAWMode(uint8_t m)      { if (m < 5) _gAWM = m; else _gAWM = AW_GLOBAL_DISABLED; }
+    inline static uint8_t getGlobalAWMode()               { return _gAWM; }
+    // 1 pin per channel on analog, 2 pins on clocked digital strips, 1 pin for the rest
+    inline static constexpr uint8_t getRequiredPins(uint8_t type)   { return IS_PWM(type) ? NUM_PWM_PINS(type) : IS_2PIN(type) + 1; }
```

Comment:
I know this may be odd for build time configuration but virtual buses use 4 "pins" for IP address.
IMO this should be reflected here as well.